velocity.py

```diff
@@ -1,33 +1,31 @@
-from tkinter import Tk
+from tkinter import *
 from tkinter import Frame
 from tkinter import LabelFrame
 from tkinter import Text
 from tkinter import Entry
 from tkinter import NORMAL
 from tkinter import DISABLED
+from tkinter.messagebox import showinfo
 from collections import deque
-#develop branch
-window = Tk()
-window.title("VELOCITY - speed + direction")
-window.geometry("600x400")
-window.minsize(width=200, height=200)
 
-text_frame = Frame(window, height=30, width=30, highlightbackground="black", highlightthickness=1)
-text_frame.pack(pady=12)
-editor_frame = LabelFrame(window, padx=2, pady=2, bd=0, text="Input your text here")
-editor_frame.pack(pady=10)
+def new_file():
+    pass
 
-text_box = Text(text_frame, height=12, width=50, padx=5, pady=5, state=NORMAL, font=("", 12))
+def open_file():
```

Comment:
changes from one.txt to two.txt not vice versa.

Reply:
Yeah thanks for this. I resolved this issue. Actually the _paragraph_(variable) was not getting updated when we were selecting the new file. I have fixed this issue.